src/update-template.js

```diff
@@ -55,6 +54,13 @@ const { TEMPLATE_STATUS_APPROVED } = require('../src/registry');
         }
 
         const savedTemplate = getFromRegistry(npmPackageMetadata.name);
+        if(adobeRecommended || savedTemplate["status"] == TEMPLATE_STATUS_APPROVED) {
```

Comment:
@AmyJZhao , we shouldn't use `adobeRecommended` for deciding if a template should be approved or not. Logic for setting up `adobeRecommended` might be changed, and that shouldn't affect the approval process. We could use a previous logic with checking a number of _stars_, a 3rd-party repo might have many stars, but that doesn't mean that we should approve that template automatically.

Reply:
I agree. From the conversations I've heard, there's still some debate on what is an "Adobe Recommended" template.